FWCore/ParameterSet/src/ParameterDescription.cc

```diff
@@ -338,34 +340,42 @@ namespace edm {
     return pset.existsAs<std::vector<ParameterSet> >(label(), isTracked());
   }
 
-  void ParameterDescription<std::vector<ParameterSet> >::writeOneElementToCfi(ParameterSet const& pset,
-                                                                              std::ostream& os,
-                                                                              int indentation,
-                                                                              bool& nextOneStartsWithAComma) {
+  void ParameterDescription<std::vector<ParameterSet> >::writeOneElementToCfi(
+      ParameterSet const& pset, std::ostream& os, int indentation, CfiOptions& options, bool& nextOneStartsWithAComma) {
     if (nextOneStartsWithAComma)
       os << ",";
     nextOneStartsWithAComma = true;
     os << "\n";
     printSpaces(os, indentation + 2);
 
+    //if (not shouldWriteUntyped(options)) {
     os << "cms.PSet(";
+    //} else {
+    //  os << "dict(";
+    //}
```

Comment:
Can these commented out parts be removed?

Reply:
done